frontend/express/public/javascripts/countly/countly.device.detail.js

```diff
@@ -196,4 +196,23 @@
         }
         return osName;
     };
+
+    countlyDeviceDetails.getOSVersionList = function(name) {
+        var codes = {};
+        var lowerCase = name.toLowerCase();
+        for (var i in countlyDeviceDetails.os_mapping) {
+            if (countlyDeviceDetails.os_mapping[i].name.toLowerCase().startsWith(lowerCase)) {
+                codes[countlyDeviceDetails.os_mapping[i].short] = true;
+            }
+            else {
+                var changed = lowerCase.replace(new RegExp("^" + countlyDeviceDetails.os_mapping[i].name.toLowerCase(), "g"), countlyDeviceDetails.os_mapping[i].short);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Found non-literal argument to RegExp Constructor](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=4839048)